painttiming.bs

```diff
@@ -62,9 +89,28 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn>Contentful</dfn>: An [=element=] or a [=pseudo-element=] counts as contentful when one or more of the following apply:
```

Comment:
nit: let's give the 'element' or 'pseudo-element' a variable name and then reference that in the bullet lists below (saying 'The element' is slightly incorrect, it would have to say 'The element or pseudo-element').